io.github.pd_l2ork.Pd_L2Ork.yml

```diff
@@ -216,7 +216,16 @@ modules:
     tag: '20241014'
     commit: 14aec94b12b51de584d12363ff6ad99b8a01be3c
   - type: file
-    # x86_64 for now only. Still waiting for aarch64.
+    # x86_64
     url: https://dl.nwjs.io/v0.67.1/nwjs-sdk-v0.67.1-linux-x64.tar.gz
     sha256: 9778278e8b935e97d613b793b029d952cde58cbd8c7c8240a246942f6ef1b945
     dest-filename: nwjs.tar.gz
+    only-arches:
+      - x86_64
+  - type: file
+    # aarch64
+    url: https://l2ork.music.vt.edu/data/pd-l2ork/nwjs-v0.60.1-linux-arm64.tar.gz
```

Comment:
did you test it? last time I tried it didn't work because of the way it was compiled...

Reply:
It works when compiled directly on Raspberry Pi, so I am hoping it will work here, as well. Waiting to see how the test goes.